apps/cow-tools/src/app/milkman/(components)/ordersTable/TransactionInfo.tsx

```diff
@@ -0,0 +1,74 @@
+import { buildBlockExplorerAddressURL } from "@bleu-fi/utils";
+import { ArrowTopRightIcon } from "@radix-ui/react-icons";
+import Link from "next/link";
+import { Address } from "viem";
+
+import {
+  decodePriceCheckerData,
+  getPriceCheckerFromAddressAndChain,
+} from "#/lib/decode";
+import { AllTransactionFromUserQuery } from "#/lib/gql/generated";
+import { priceCheckersArgumentsMapping } from "#/lib/priceCheckersMappings";
+import { truncateAddress } from "#/utils/truncate";
+
+export function TransactionInfo({
+  order,
+}: {
+  order: AllTransactionFromUserQuery["users"][0]["transactions"][0]["swaps"][0];
+}) {
+  const priceChecker = getPriceCheckerFromAddressAndChain(
+    order.chainId as 5,
```

Comment:
is this right?

Reply:
I have a task to change it all to multiple chains. At the end of this week, I will have to solve this for one issue.